kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/splash/presenter/ISplashPresenter.java

```diff
@@ -1,13 +1,25 @@
 package com.kin.ecosystem.splash.presenter;
 
+import android.support.annotation.IntDef;
 import com.kin.ecosystem.base.IBasePresenter;
 import com.kin.ecosystem.splash.view.ISplashView;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
 public interface ISplashPresenter extends IBasePresenter<ISplashView> {
 
-    void getStartedClicked();
+	int TRY_AGAIN = 0x00000001;
```

Comment:
I think TRY_AGAIN and SOMETHING_WENT_WRONG should be part of the view interface

Reply:
nope, the presenter shouldn't have any view related objects.